### ISSUE TITLE
Code highlighting breaks because of the language header

### ISSUE THREAD

**Issue by tkapustina** [SpecFlowOSS/SpecFlow.Rider#138]
STR:
1. Open a feature file
2. Add `# language:  `

ACTUAL:
![image](https://user-images.githubusercontent.com/48007045/121495581-66d59f80-c9e2-11eb-910e-43d0ff4b8230.png)
[https://youtrack.jetbrains.com/issue/DEXP-592129](https://youtrack.jetbrains.com/issue/DEXP-592129
)
```
Incremental reparse Index and length must refer to a location within the string. Parameter name: length
JetBrains Rider 2021.2 Build RD-212.4047.84 Date 2021-06-10T05:39:18.045+0000

--- EXCEPTION #1/3 [ArgumentOutOfRangeException]
Message = “Index and length must refer to a location within the string.”
ExceptionPath = Root.InnerException.InnerException
ClassName = System.ArgumentOutOfRangeException
HResult = COR_E_ARGUMENTOUTOFRANGE=80131502
Source = mscorlib
ParamName = length

StackTraceString = “
at System.String.Substring(Int32 startIndex, Int32 length)
at ReSharperPlugin.SpecflowRiderPlugin.Psi.GherkinKeywordProvider.GetKeywordsList(String language) in D:\a\SpecFlow.Rider\SpecFlow.Rider\src\dotnet\ReSharperPlugin.SpecflowRiderPlugin\Psi\GherkinKeywordProvider.cs:line 76
at ReSharperPlugin.SpecflowRiderPlugin.Psi.GherkinKeywordProvider.GetAllKeywords(String language) in D:\a\SpecFlow.Rider\SpecFlow.Rider\src\dotnet\ReSharperPlugin.SpecflowRiderPlugin\Psi\GherkinKeywordProvider.cs:line 31
at ReSharperPlugin.SpecflowRiderPlugin.Psi.GherkinLexer.UpdateLanguage(String language) in D:\a\SpecFlow.Rider\SpecFlow.Rider\src\dotnet\ReSharperPlugin.SpecflowRiderPlugin\Psi\GherkinLexer.cs:line 56
at ReSharperPlugin.SpecflowRiderPlugin.Psi.GherkinLexer.Advance() in D:\a\SpecFlow.Rider\SpecFlow.Rider\src\dotnet\ReSharperPlugin.SpecflowRiderPlugin\Psi\GherkinLexer.cs:line 174
at JetBrains.ReSharper.Psi.Parsing.TokenBuffer.ReScanUpToEnd(ILexer lexer)
at JetBrains.ReSharper.Psi.Files.CachedPsiFile.GetIncrementalReparseResult(TokenBuffer oldTokenBuffer, IBuffer newBuffer, ISecondaryRangeTransla
```

**Comment by tkapustina** [comment]
One more example
![image](https://user-images.githubusercontent.com/48007045/121495354-3aba1e80-c9e2-11eb-8004-dd0c363a18f5.png)

**Comment by nemesv** [comment]
Fixed in version 1.8.0.

**Comment by tkapustina** [comment]
Verified.
![image](https://user-images.githubusercontent.com/48007045/122380552-abc47d80-cf70-11eb-9fca-a929a4639ddb.png)
I'm not sure that language header should be highlighted too, because it's a proper line. 
And it would be cool to suggest a quick fix to change the language.
